packages/esm-patient-vitals-app/src/vitals/paginated-vitals.component.tsx

```diff
@@ -13,12 +15,13 @@ import {
 import { useLayoutType, usePagination } from '@openmrs/esm-framework';
 import { PatientChartPagination } from '@openmrs/esm-patient-common-lib';
 import styles from './paginated-vitals.scss';
+
 interface PaginatedVitalsProps {
   pageSize: number;
   pageUrl: string;
   urlLabel: string;
-  tableRows: Array<Record<string, string>>;
-  tableHeaders: Array<Record<string, string | boolean>>;
+  tableRows: Array<typeof DataTableRow>;
```

Comment:
Technically, it's probably better to ts-ignore these for now... They are types, the types just aren't visible to TS.